From: Ubuntu Core Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 14 Feb 2019 18:59:14 -0800
Subject: tkinter-import

# DP: suggest installation of python-tk package on failing _tkinter import
# DP: suggest installation of python-tk package on failing _tkinter import
---
 Lib/tkinter/__init__.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Lib/tkinter/__init__.py b/Lib/tkinter/__init__.py
index 4955c9e..090a555 100644
--- a/Lib/tkinter/__init__.py
+++ b/Lib/tkinter/__init__.py
@@ -33,7 +33,10 @@ tk.mainloop()
 import enum
 import sys
 
-import _tkinter # If this fails your Python may not be configured for Tk
+try:
+    import _tkinter
+except ImportError as msg:
+    raise ImportError(str(msg) + ', please install the python3-tk package')
 TclError = _tkinter.TclError
 from tkinter.constants import *
 import re
